src/controllers/employer.ts

```diff
@@ -0,0 +1,96 @@
+import { Request, Response, NextFunction } from 'express'
+import bcrypt from 'bcrypt'
+import passport from 'passport'
+
+import {
+  NotFoundError,
+  UnauthorizedError,
+  InternalServerError,
+  BadRequestError,
+} from '../helpers/apiError'
+import Employer from '../entities/Employer.postgres'
+import Credential from '../entities/Credential.postgres'
+import JobPost from '../entities/JobPost.postgres'
+
+//**Auth controllers */
+export const localLogin = async (
+  req: Request,
+  res: Response,
+  next: NextFunction
+) => {
+  passport.authenticate('local', function (error, user: Employer, info) {
+    if (error) {
+      return next(new InternalServerError())
+    }
+    if (!user) {
+      if (info.message === 'Invalid email or password') {
+        return next(new UnauthorizedError(info.message))
+      }
+      return next(new NotFoundError(info.message))
+    }
+
+    res.send(user)
```

Comment:
Your login test failed because after login, the controller send back a user object in the response body. But you were checking a message, which did not exist on the response body